libraries/chain/include/eosio/chain/chain_config.hpp

```diff
@@ -15,6 +15,7 @@ namespace eosio { namespace chain {
  */
 struct chain_config_v0 {
 
+   //order must match parameters as ids are used in serialization
```

Comment:
Add an additional comment: do not add more to this list; doing so will break chain_config_v1

Reply:
Never mind. It's already implied by the v0/v1 inheritance.